cmake/linker/mwdt/target.cmake

```diff
@@ -0,0 +1,203 @@
+# SPDX-License-Identifier: Apache-2.0
+
+find_program(CMAKE_LINKER ${CROSS_COMPILE}ldac PATH ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
```

Comment:
It's non-standard to detect the linker this late isn't it?

Let's align with the other toolchains.